CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.5.2
+
+<!-- generated comparing v5.5.1..master -->
+
+_Mar 21, 2022_
+
+A big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🛠 Dragging on disabled sliders is fixed by @mnajdova (#31882)
```

Comment:
This was an issue only for the base (unstyled) slider, I don't think we should highlight it, as it is not affecting lot of the audience.